infotv/frontend/package.json

```
@@ -16,30 +16,30 @@
     "private": true,
     "type": "module",
     "devDependencies": {
-        "@types/lodash": "^4.14.170",
-        "@types/react": "^17.0.11",
-        "@types/react-dom": "^17.0.8",
-        "@types/react-transition-group": "1.1.6",
-        "@typescript-eslint/eslint-plugin": "^4.28.0",
-        "@typescript-eslint/parser": "^4.28.0",
-        "esbuild": "^0.16.8",
-        "eslint": "^7.29.0",
-        "eslint-plugin-react": "^7.24.0",
+        "@types/lodash": "^4.14.191",
+        "@types/react": "^18.0.28",
+        "@types/react-dom": "^18.0.11",
+        "@types/react-transition-group": "4.4.5",
```

```suggestion
        "@types/react-transition-group": "1.1.8",
```

Needs update of yarn.lock afterwards